### PR TITLE
Bugfixy 5

### PR DESCRIPTION
**Pull request by Bod9001**
### Changelog:
CL: [Fix] fixes a potential game freeze from blob
CL: [Fix] Fixes the nuclear disk teleporting around randomly
CL: [Fix] fixes overdose instantly killing you
CL: [Fix] fixes Getting kicked from EmoteAction
CL: [Fix] fixes Hopefully fixes client synchronised Effects showing at wrong time

### CODE REVIEW

File: UnityProject/Assets/StreamingAssets/Maps/MainStations/BoxStationV1.json [hunk 1]
```diff
@@ -362183,6 +362183,12 @@
               ]
             }
           },
+          {
+            "GitID": "CD2uUGXjPEuvXGheerXA_0┼0┼0┼",
+            "PrefabID": "CD2uUGXjPEuvXGheerXA",
+            "Name": "MainStation",
```
Comment (corp-0) [inline UnityProject/Assets/StreamingAssets/Maps/MainStations/BoxStationV1.json:362189]: is this the name of the gameobject that has the marker component? Can it be something else than ``MainStation``?

Comment (Bod9001) [reply UnityProject/Assets/StreamingAssets/Maps/MainStations/BoxStationV1.json:362189]: yeah, just couldn't think of a good name for marking what  makes the main sstation

Comment (corp-0) [reply UnityProject/Assets/StreamingAssets/Maps/MainStations/BoxStationV1.json:362189]: yeah but does it have to be that or could the ``Name`` field have the actual name of the station? We need a field with the name of the station because this is not good 
image

Comment (Bod9001) [reply UnityProject/Assets/StreamingAssets/Maps/MainStations/BoxStationV1.json:362189]: the issue as we have to re-factor it since it was loading by scene name, Would have to set up the thing to get the display name instead of the filename

File: UnityProject/Assets/Scripts/HealthV2/Living/Damage/Trauma/TraumaTypes/TraumaBurning.cs [hunk 2]
```diff
@@ -80,7 +80,7 @@ public override void ProgressDeadlyEffect()
 		/// </summary>
 		private void AshBodyPart()
 		{
-
+			if (bodyPart.BodyPartType == BodyPartType.Head) return;
```
Comment (MaxIsJoe) [inline UnityProject/Assets/Scripts/HealthV2/Living/Damage/Trauma/TraumaTypes/TraumaBurning.cs:83]: why?
Heads should be able to burn and melt.

Comment (corp-0) [reply UnityProject/Assets/Scripts/HealthV2/Living/Damage/Trauma/TraumaTypes/TraumaBurning.cs:83]: and what is this fixing? It isn't mentioned anywhere

File: UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs [hunk 3]
```diff
@@ -23,7 +23,7 @@ public class BodyHealthEffect : MetabolismReaction
 	public bool CanOverdose = true;
 
 	[ShowIf(nameof(CanOverdose))] public float ConcentrationBloodOverdose = 20f;
-	[ShowIf(nameof(CanOverdose))] public float OverdoseDamageMultiplier = 1;
+	[ShowIf(nameof(CanOverdose))] public float OverdoseDamageMultiplierNew = 1;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs:26]: why did you rename this?

File: UnityProject/Assets/Scripts/HealthV2/Living/Damage/Trauma/TraumaTypes/TraumaBurning.cs (hunk 2 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/HealthV2/Living/Damage/Trauma/TraumaTypes/TraumaBurning.cs:83]: so, it causes your entire body to basically get removed by flames

File: UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs (hunk 3 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs:26]: part of OD balancing

Comment (corp-0) [reply UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs:26]: ok but how changing the name and nothing else about it is balancing OD?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs:26]: it resets the multiplier on the SOs

**Comment by craftxbox** [comment]
Does this fix the hub reporting filepaths for station names?

**Comment by corp-0** [comment]
> Does this fix the hub reporting filepaths for station names?

I asked the same thing. It doesn't

File: UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs (hunk 3 above)
Comment (corp-0) [reply UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs:26]: that is kind of eck ngl
